lib/fibers_stack/SFibersStackHitFinderPar.cc

```diff
@@ -32,6 +32,62 @@ void SFibersStackHitFinderPar::clear()
 {
     fA0 = 0.0;
     fLambda = 0.0;
+    fResPos.Reset();
+    fResEne.Reset();
+}
+/**
+ * The resolution equations are typically f(x) = p0/x + p1/x^{1/2} + p2/x^{3/2}
+ * from Poisson statistics.
+ * Here we can set/get the number of resolution parameters (default=3) and the
+ * parameters themselves
+ */
+UInt_t SFibersStackHitFinderPar::getNumOfResPars() const
+{
+	return numOfResPars;
+}
+
+Float_t SFibersStackHitFinderPar::getResPos(UInt_t i) const
+{
+	Float_t ret = 0.;
+	if(i < fResPos.GetSize() ) {
+		ret = fResPos[i];
+	} else {
+        	std::cerr << "TArrayF index is more than allowed in numOfResPars" << std::endl;
+		return 0.;
+	}
+	return ret;
+}
+
+Float_t SFibersStackHitFinderPar::getResEne(UInt_t i) const
+{
+	Float_t ret = 0.;
+	if(i < fResEne.GetSize() ) {
+		ret = fResEne[i];
+	} else {
+        	std::cerr << "TArrayF index is more than allowed in numOfResPars" << std::endl;
+		return 0.;
+	}
+	return ret;
+}
+
+void SFibersStackHitFinderPar::setNumOfResPars(UInt_t i) {
```

Comment:
This requires that Set() method of TArrayF is called otherwise it may break the set/get parameter operation. But anyway, the number of params is fixed and should not be changed.